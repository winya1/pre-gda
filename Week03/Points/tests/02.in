9
0 0 1 1
1 1 0 0
-1 -1 1 0
5 0 0 6
6 0 0 5
6 0 5 5
5 5 0 7
5 5 8 0
2147483647 -2147483648 -2147483647 -2147483647
