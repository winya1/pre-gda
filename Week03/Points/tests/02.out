Case #1: 0 0
Case #2: 0 0
Case #3: 1 0
Case #4: 5 0
Case #5: 0 5
Case #6: 6 0
Case #7: 0 7
Case #8: 5 5
Case #9: -2147483647 -2147483647
